src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapPlot.jsx

```diff
@@ -37,13 +38,26 @@ const HeatmapPlot = (props) => {
   const properties = useSelector((state) => state.cellSets.properties);
   const hidden = useSelector((state) => state.cellSets.hidden);
 
+  const heatmapSettings = useSelector((state) => state.componentConfig.interactiveHeatmap?.config) || {};
```

Comment:
We should have `interactiveHeatmap` assigned to a variable - it is not great to have it hardcoded like this. This `interactiveHeatmap` sounds to me more like component type, but we already have `const componentType = 'Heatmap';` on line 26. I am thinking that these two have to be the same thing actually -- how about we make `const componentType = 'InteractiveHeatmap';` and then we use `componentType` instead of this hardcoded value?

Reply:
yeah, i was thinking of a way to do this properly, and considered basically what you described. will edit.